packages/dds/tree/src/feature-libraries/sequence-field/utils.ts

```diff
@@ -333,17 +474,19 @@ export function tryExtendMark(
         }
         case "Delete": {
             const lhsDetach = lhs as Detach;
-            if (rhs.tomb === lhsDetach.tomb) {
-                lhsDetach.count += rhs.count;
-                return true;
-            }
-            break;
+            lhsDetach.count += rhs.count;
```

Comment:
Why don't we check `tomb` anymore?

Reply:
Right, I'm removing `tombs`, which would better be done in a separate cleanup PR. I'll spin that up.